tests/e2e/specs/wcpay/merchant/merchant-progressive-onboarding.spec.js

```diff
@@ -0,0 +1,146 @@
+/**
+ * External dependencies
+ */
+const { merchant, evalAndClick } = require( '@woocommerce/e2e-utils' );
+
+/**
+ * Internal dependencies
+ */
+import { merchantWCP, uiLoaded } from '../../../utils';
+
+describe( 'Admin merchant progressive onboarding', () => {
+	beforeAll( async () => {
+		await merchant.login();
+		await merchantWCP.enableProgressiveOnboarding();
+		await merchantWCP.enableActAsDisconnectedFromWCPay();
+		// todo clear/mock wcpaySettings.onboardingFlowState = null, so no previous session is restored
```

Comment:
to be super-safe we need to reset it because it test fails at some moment then it'll restore the state and the test will be broken forever